src/state/slices/marketDataSlice/marketDataSlice.tsx

```diff
@@ -128,6 +141,22 @@ export const marketApi = createApi({
         try {
           const currentMarketData = await findByAssetId({ assetId })
           if (!currentMarketData) throw new Error()
+
+          //FOXy specific api call to retrieve max supply
+          if (assetId === FOXY_ASSET_ID) {
```

Comment:
popping a requested change on here until this is removed and we get this from the market service